app/javascript/controllers/team_controller.js

```diff
@@ -0,0 +1,50 @@
+
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = ["team_count", "modal_user", "team"];
```

Comment:
це уже не рубі, тут треба дотримуватися свого неймінг конвеншина
```suggestion
  static targets = ["teamCount", "modalUser", "team"];
```